hermann.gemspec

```diff
@@ -24,10 +24,12 @@ Gem::Specification.new do |s|
   s.rubygems_version = '2.2.2'
   s.specification_version = 3 if s.respond_to?(:specification_version)
 
-  s.add_dependency 'concurrent-ruby', '~> 0.7.0'
+  s.add_dependency 'json', '~> 1.8.2'
   s.add_dependency 'thread_safe', '~> 0.3.4'
```

Comment:
thread_safe isn't only required for Java-specific code. it's used in `hermann/producer` before we get into platform specific implementations